source/manage-apps/deploy.txt

```diff
@@ -22,6 +22,7 @@ Application Deployment
    Deployment Models & Regions </manage-apps/deploy/deployment-models-and-regions>
    Deployment Draft </manage-apps/deploy/deployment-draft>
    Disable Deployment Drafts </manage-apps/deploy/disable-deployment-drafts>
+   Compact an Atlas Volume </manage-apps/deploy/compact-disk>
```

Comment:
Seems weird to group this under Deployment - IMO it feels more like something we'd have under MongoDB Data Sources or Sync

Reply:
Yep. I struggled to find the best home for it. Moved it under the (now linked) page on compacting.